api/CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ All notable changes to the **Prowler API** are documented in this file.
 
 ---
 
+## [v1.5.1] (Prowler v5.4.1)
```

Comment:
We need to update the API version. Do it here or in the other PR you have opened.